osgar/drivers/sicklidar.py

```diff
@@ -24,10 +24,12 @@ def __init__(self, config, bus):
     def parse_raw_data(raw_data):
         """Parse scan data for TiM571 and TiM551 SICK LIDARs"""
         data = raw_data.split()
-        assert len(data) in [854, 846, 583, ], len(data)
+        assert len(data) in [26, 854, 846, 583, 1663], len(data)
         assert data[1] == b'LMDscandata', data[:2]
         timestamp = int(data[9], 16)  # TODO verify
         freq = int(data[16], 16)
+        if len(data) == 26:  # empty scan
+            return [], []
```

Comment:
Here remission is [] but later on it is None. We should pick just one value to represent the fact that remission data is not present. What do you think?

Reply:
see the note above